crates/locksmith/Cargo.toml

```diff
@@ -18,3 +18,13 @@ lazy_static = "=1.2.0"
 log = "=0.4.8"
 parking_lot ="=0.9.0"
 snowflake = "=1.3.0"
+
+[features]
+#default brings in no features
+default =[]
+
+#this feature allows the compilation with backtraces involved
+sync_backtrace = []
```

Comment:
in tracker.rs, it's `sync_backtrace_capture`

Reply:
good catch error during renaming :)